README.md

```diff
@@ -37,4 +37,4 @@ To accomodate JNA, add the following to the consuming app's proguard rules:
 -dontwarn java.awt.*
 -keep class com.sun.jna.* { *; }
 -keepclassmembers class * extends com.sun.jna.* { public *; }
-```
+```
```

Comment:
What changed?

Reply:
Not sure.. I had a change but later removed it. It might just be whitespace but it's not clear from the diff.